Compile warning about unsigned/signed comparsion

[ 65%] Building CXX object paddle/fluid/inference/tensorrt/convert/CMakeFiles/test_trt_activation_op.dir/io_converter.cc.o
In file included from /paddle/paddle/fluid/inference/tensorrt/convert/activation_op.cc:15:0:
/paddle/paddle/fluid/inference/tensorrt/convert/op_converter.h: In member function ‘void paddle::inference::tensorrt::OpConverter::ConvertBlock(const paddle::framework::proto::BlockDesc&, paddle::inference::tensorrt::TensorRTEngine*)’:
/paddle/paddle/fluid/inference/tensorrt/convert/op_converter.h:52:26: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     for (size_t i = 0; i < block.ops_size(); i++) {
                          ^
In file included from /paddle/paddle/fluid/inference/tensorrt/convert/test_activation_op.cc:20:0:
/paddle/paddle/fluid/inference/tensorrt/convert/op_converter.h: In member function ‘void paddle::inference::tensorrt::OpConverter::ConvertBlock(const paddle::framework::proto::BlockDesc&, paddle::inference::tensorrt::TensorRTEngine*)’:
/paddle/paddle/fluid/inference/tensorrt/convert/op_converter.h:52:26: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     for (size_t i = 0; i < block.ops_size(); i++) {


In file included from /paddle/paddle/fluid/inference/tensorrt/convert/mul_op.cc:15:0:
/paddle/paddle/fluid/inference/tensorrt/convert/op_converter.h: In member function ‘void paddle::inference::tensorrt::OpConverter::ConvertBlock(const paddle::framework::proto::BlockDesc&, paddle::inference::tensorrt::TensorRTEngine*)’:
/paddle/paddle/fluid/inference/tensorrt/convert/op_converter.h:52:26: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     for (size_t i = 0; i < block.ops_size(); i++) {
                          ^
Scanning dependencies of target save_load_combine_op_test
[ 61%] Building CXX object paddle/fluid/operators/CMakeFiles/save_load_combine_op_test.dir/save_load_combine_op_test.cc.o
In file included from /paddle/paddle/fluid/inference/tensorrt/convert/conv2d_op.cc:15:0:
/paddle/paddle/fluid/inference/tensorrt/convert/op_converter.h: In member function ‘void paddle::inference::tensorrt::OpConverter::ConvertBlock(const paddle::framework::proto::BlockDesc&, paddle::inference::tensorrt::TensorRTEngine*)’:
/paddle/paddle/fluid/inference/tensorrt/convert/op_converter.h:52:26: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     for (size_t i = 0; i < block.ops_size(); i++) {

Scanning dependencies of target top_k_op
In file included from /paddle/paddle/fluid/inference/tensorrt/convert/test_op_converter.cc:17:0:
/paddle/paddle/fluid/inference/tensorrt/convert/op_converter.h: In member function ‘void paddle::inference::tensorrt::OpConverter::ConvertBlock(const paddle::framework::proto::BlockDesc&, paddle::inference::tensorrt::TensorRTEngine*)’:
/paddle/paddle/fluid/inference/tensorrt/convert/op_converter.h:52:26: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     for (size_t i = 0; i < block.ops_size(); i++) {

--- a/paddle/fluid/inference/tensorrt/convert/op_converter.h
+++ b/paddle/fluid/inference/tensorrt/convert/op_converter.h
@@ -49,7 +49,7 @@ class OpConverter {
   // convert fluid block to tensorrt network
   void ConvertBlock(const framework::proto::BlockDesc& block,
                     TensorRTEngine* engine) {
-    for (size_t i = 0; i < block.ops_size(); i++) {
+    for (int i = 0; i < block.ops_size(); i++) {
       const auto& op = block.ops(i);
       OpConverter::Run(op, engine);
     }

